CopilotKit/packages/react-core/tsup.config.ts

```diff
@@ -5,7 +5,6 @@ export default defineConfig((options: Options) => ({
   format: ["esm", "cjs"],
   dts: true,
   minify: false,
-  clean: true,
```

Comment:
dang. good find.

Reply:
Down the line it might be a good idea to refactor the clean config variable in the `tsup.config.ts` itself, e.g. via an environment variable. 

adding `--clean` everywhere is a bit brittle.

But we can address that later, i will create an issue

Reply:
yeah the best way to approach this IMO is to have a base "tsup.config.ts" and "tsup.config.dev.ts" file that sits in a shared config folder, imported by the libs and extended.

Reply:
yes, added this issue here, with your approach as an alternative